lib/loggers/dev-logger.js

```diff
@@ -2,7 +2,7 @@
 
 const path = require('path')
 const color = require('../color')
-
+const devip = require('dev-ip')()
```

Comment:
I'm not a fan of depending a small package that we can easily implement as a util. Can we include the logic in houl?